includes/class-wc-payments-webhook-processing-service.php

```diff
@@ -135,10 +135,11 @@ public function process( array $event_body ) {
 		// Extract information about the webhook event.
 		$event_type = $this->read_webhook_property( $event_body, 'type' );
 
-		Logger::debug( 'Webhook received: ' . $event_type );
 		Logger::debug(
-			'Webhook body: '
-			. var_export( WC_Payments_Utils::redact_array( $event_body, WC_Payments_API_Client::API_KEYS_TO_REDACT ), true ) // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_export
+			'< WEBHOOK RECEIVED: ' . $event_type,
+			[
+				'body' => var_export( WC_Payments_Utils::redact_array( $event_body, WC_Payments_API_Client::API_KEYS_TO_REDACT ), true ), // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_var_export
```

Comment:
```suggestion
				'body' => WC_Payments_Utils::redact_array( $event_body, WC_Payments_API_Client::API_KEYS_TO_REDACT ),
```

Now that this string goes to the context, it makes the output very weird. I think we can skip it.

<img width="1223" alt="Screenshot 2025-04-08 at 20 07 27" src="https://github.com/user-attachments/assets/8bda9056-2b7b-4ba0-83ad-3e150b4125d6" />

Reply:
Thanks for checking this flow and noticing it! Addressed both comments in 76e92eb8b